case-lib/lib.sh

```diff
@@ -1030,10 +1030,20 @@ re_enable_ntp_sync()
     sudo timedatectl set-ntp true
 }
 
+# Get alsa-info of DUTs in CI
+print_alsa_info()
+{
+    OUTPUT_FILE="/tmp/alsa-info.txt"
+    alsa-info --output "$OUTPUT_FILE" --with-aplay --with-amixer --with-alsactl --with-configs
+    ALSA_INFO_TEXT=$(cat "$OUTPUT_FILE")
+    dlogi "$ALSA_INFOTEXT"
```

Comment:
This is convoluted. Just do `alsa-info --stdout ...` and you can drop all the other lines. In fact, you can even drop the function and inline the call.